### PR TITLE
New Drone Manufacturer Lore

### PR DESCRIPTION
**Pull request by avabee**
Taking inspiration from Trillby's lore, I decided to write my own interpretation of how drones are constructed and behave, based off of my own (limited) experience and understanding of the field.

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
How dare you be better at this then me!
PS their is miner drones, and hive bots are a drone type.

File: lore/technology/drone_manufacturer.mediawiki [hunk 1]
```diff
@@ -0,0 +1,16 @@
+=History=
+Due to advancements in the field of macro-scale 3D printing after the field of nanotechnology was largely abandoned due to numeous scandals, manufacturing delicate, yet strong metal drone casings could be done reliably in an automated assembly process. Using this as a starting point, NanoTrasen decided to devote a substantial portion of their research and development facilities to achieving the goal of fully-automated maintenance and construction robots by 2520.
```
Comment (CameronWoof) [inline lore/technology/drone_manufacturer.mediawiki]: "... due to nume**r**ous scandals..."

File: lore/technology/drone_manufacturer.mediawiki [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+The largest issue with designing the drones proved to be commonsense reasoning, as while construction was a relatively simple process, the sensorimotor component of the drones proved elusive. At the time, NanoTrasen drones were capable of complex reasoning, conversation, calculation, and planning, but failed to provide human-like performance on tasks like image recognition and movement.
```
Comment (CameronWoof) [inline lore/technology/drone_manufacturer.mediawiki]: "... to be common sense reasoning..."

The second sentence doesn't seem related to the first, and seems like a weird order of events. Image recognition and movement are the parts of robotics that we're working most closely with *now*. Why is it the last thing that NanoTrasen managed to figure out?

I'd change the failure to be in responding naturally to physical stimulus, or some core behavior that requires sensorimotor components.

**Review by CameronWoof — CHANGES_REQUESTED** [review]

File: lore/technology/drone_manufacturer.mediawiki (hunk 2 above)
Comment (avabee) [reply lore/technology/drone_manufacturer.mediawiki]: Image recognition is what we're working with now, but we SUCK at it. Majorly so. Essentially: We think it works good, but it don't. https://medium.com/bethgelab/neural-networks-seem-to-follow-a-puzzlingly-simple-strategy-to-classify-images-f4229317261f

File: lore/technology/drone_manufacturer.mediawiki (hunk 1 above)
Comment (avabee) [reply lore/technology/drone_manufacturer.mediawiki]: I was trying to reference the Grey Tide from Polaris' lore, but I forgot the details surrounding it and didn't know if it was canon anyway.

Comment (CameronWoof) [reply lore/technology/drone_manufacturer.mediawiki]: I'm pointing out a typo.

**Pull request merged**